lib/home/profile/report/issue_report.dart

```diff
@@ -52,51 +50,32 @@ class _IssueReportScreenState extends State<IssueReportScreen> {
                     labelText: 'name',
                     labelStyle: style,
                     floatingLabelStyle: style,
-                    border: MaterialStateOutlineInputBorder.resolveWith((states) {
-                      return OutlineInputBorder(
-                        borderSide: states.isFocused
-                            ? BorderSide(color: colors.primary, width: 2)
-                            : BorderSide(color: colors.onBackground.withOpacity(0.5)),
-                      );
-                    }),
+                    border: border,
                   ),
                   validator: (value) {
                     if (value?.isEmpty ?? true) {
                       return 'Please enter your name';
                     }
                     return null;
                   },
-                  onSaved: (value) {
-                    _name = value ?? '';
-                  },
+                  onSaved: (value) => _name = value ?? '',
                 ),
               ),
               Padding(
                 padding: const EdgeInsets.only(bottom: 16.0),
-                // Add bottom padding here
                 child: TextFormField(
                   decoration: InputDecoration(
                     isDense: true,
                     labelText: 'email',
                     labelStyle: style,
                     floatingLabelStyle: style,
-                    border: MaterialStateOutlineInputBorder.resolveWith((states) {
-                      return OutlineInputBorder(
-                        borderSide: states.isFocused
-                            ? BorderSide(color: colors.primary, width: 2)
-                            : BorderSide(color: colors.onBackground.withOpacity(0.5)),
-                      );
-                    }),
+                    border: border,
                   ),
-                  validator: (value) {
-                    if (value?.isEmpty ?? true) {
-                      return 'Please enter a valid email address';
-                    }
-                    return null;
-                  },
-                  onSaved: (value) {
-                    _email = value ?? '';
+                  validator: (value) => switch (value) {
+                    final email? when EmailValidator.validate(email) => null,
+                    _ => 'Please enter a valid email address',
                   },
```

Comment:
I added an email validator package a while back, so I thought it'd be nice to use that here.